shell/initialize/plugins.js

```diff
@@ -52,3 +53,62 @@ export async function installPlugins(app, inject, Vue) {
   // Order matters here. This is coming after the other plugins specifically so $cookies can be installed. i18n/init relies on prefs/get which relies on $cookies.
   Vue.use(i18n, { store: app.store });
 }
+
+/**
+ * Injects a key-value pair into a given context and installs a Vue plugin with
+ * the provided key-value pair.
+ *
+ * @param {string} key The key to be injected
+ * @param {*} value The value to be injected
+ * @param {object} context The context object into which the key-value pair will
+ * be injected
+ * @param {object} Vue The Vue instance
+ * @returns {void}
+ *
+ * @property {object} context.app The app object within the context
+ * @property {object} context.store The store object within the context
+ *
+ * Note: This function mutates the context object, including the context app and
+ * store.
+ */
+function inject(key, value, context, Vue) {
```

Comment:
Because the context and vue instance are both present in this app, would it instead make sense to pass and inject function to the plugins that already have these arguments captures and allow the plugin to just pass the key and value?

Do we want the plugins to be able to change the instance and context?

Reply:
Yeah this is something I was thinking as well, to just export something to be imported in the entry file or expose the instance (and config?) 
I would find more easy to finally install, import and integrate anything inside the same place, plus I am not so sure how much is easy the other way around.